addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2524,6 +2550,28 @@ export class OdooEditor extends EventTarget {
                 this.historyStep();
             }
         }
+
+        // handle stars
+        if (node.nodeType === Node.ELEMENT_NODE && node.className.includes('fa-star')) {
```

Comment:
Shouldn't this code be protected to only affect your new snippets ? Or maybe I'm missing something ?

Reply:
Yeah you're right. I guess I was blinded by the fact that I didn't support the idea of that part of the feature :p